src/platform/Tizen/Logging.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <platform/logging/LogV.h>
+
+#include <core/CHIPConfig.h>
+#include <support/logging/Constants.h>
+
+#include <dlog.h>
+#include <stdio.h>
+
+#ifdef LOG_TAG
+#undef LOG_TAG
+#endif
+#define LOG_TAG "CHIP"
+
+#ifndef LOGD
+#define LOGD(fmt, args...) dlog_print(DLOG_DEBUG, LOG_TAG, fmt, ##args)
+#endif
+#ifndef LOGE
+#define LOGE(fmt, args...) dlog_print(DLOG_ERROR, LOG_TAG, fmt, ##args)
+#endif
+#ifndef LOGI
+#define LOGI(fmt, args...) dlog_print(DLOG_INFO, LOG_TAG, fmt, ##args)
+#endif
```

Comment:
Why separate macros that are  used only once in the LogV method?

Since this is a single CPP file (not included to differ by TAG), the several layers of indirections make the code more complex than it should.

Please write in the CPP file directly things like

```cpp
dlog_print(DLOG_INFO, kChipTag, "%s: %s", module, msgBuf);
```

Reply:
Done.